src/NotificationTemplate.php

```diff
@@ -613,7 +613,7 @@ public function getDataToSend(NotificationTarget $target, $tid, $to, array $user
         $mailing_options['items_id']     = method_exists($target->obj, "getField")
          ? $target->obj->getField('id')
          : 0;
-        if (isset($target->obj->documents)) {
+        if (property_exists($target, 'documents') && isset($target->obj->documents)) {
```

Comment:
Pr closed, but I don't understand this change

Should the part added be `property_exists($target, 'obj')`, no ? @trasher

Reply:
@AdrienClairembault sorry (missping)

Reply:
I think there is indeed a mistake and it should be `if (property_exists($target->obj, 'documents')`